src/Components/Shared/Input/index.js

```diff
@@ -0,0 +1,12 @@
+import Styles from './input.module.css';
+
+const Input = ({ type, name, value, onChange }) => {
+  return (
+    <label className={Styles.label}>
+      {name}
+      <input type={type} name={name} value={value} onChange={onChange} />
```

Comment:
```suggestion
      <input type={type} name={name} value={value} onChange={onChange} required />
```
Add required to the input

Reply:
Thanks for the comment, I just fixed it.